### DISCUSSION TITLE
Can't reexport the named export 'ReactFlow' from non EcmaScript module (only default export is available)

### DISCUSSION THREAD

**Discussion by kiranlpatil** [xyflow/xyflow#3337] [Q&A] [answered]
I have used ReactFlow into one of my problem and published it onto private npm repository. Now after installing that npm repo into one of my other repo and using the published component which has ReactFlow library, it's giving me error as this below:

Tech stack are :: ReactJs 18, Typescript.  

```
Failed to compile.

./node_modules/reactflow/dist/esm/index.mjs
Can't reexport the named export 'ReactFlow' from non EcmaScript module (only default export is available)
```

**Comment by moklick** [comment, marked as answer]
could you try 11.8.2?
